cdap-ui/app/features/datasets/templates/detail.html

```diff
@@ -15,20 +15,20 @@
 -->
 
 <my-breadcrumbs></my-breadcrumbs>
-<div class="row">
+<div class="row type-heading">
```

Comment:
Is it possible for us to use ```program-heading``` tag just like in programs? Any specific reason why we are using css class ```type-heading``` for datasets and streams alone ? May be we should change ```program-heading``` to ```cdap-object-heading```?